packages/esm-patient-clinical-view-app/src/clear-workspace.ts

```diff
@@ -0,0 +1,5 @@
+import { detachAll } from '@openmrs/esm-framework';
+
+export const clearWorkspace = (extensionSlotName) => {
```

Comment:
The same here. Generally though, if you have two functions doing exactly the same you can move them to `esm-patient-common-lib`. Then you don't have to copy/paste them. But again, in this specific case I'd just remove them entirely.